vars/isPypiPublishable.groovy

```diff
@@ -11,5 +11,5 @@ def call() {
     Utils utils = new Utils(this)
 
     def tags = gitUtils.isCommitTagged()
-    return utils.isPublishable(tags)
+    return utils.isPublishable(BRANCH_NAME, tags)
```

Comment:
Is there a test that (would have) caught this? Should there be?

Reply:
So there's a unit testing framework called [Jenkins Pipeline Unit](https://github.com/jenkinsci/JenkinsPipelineUnit) that tests the actual steps rather than the classes that support the steps.  That would have caught it.  The implementation of this testing framework will take a lot of work and will be its own issue.